changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 24th September 2024
+* Updated response model of the operation [Get cancellation policies by reservations](../operations/cancellationpolicies.md#get-cancellation-policies-by-reservations).

```

Comment:
This should describe what exactly has been change, but since the endpoint is new and restricted, maybe we don't have to mention it and just change it. @MikeAdamsMews thoughts?